rxgroups-android/src/main/java/com/airbnb/rxgroups/android/ObservableTagFactory.java

```diff
@@ -0,0 +1,5 @@
+package com.airbnb.rxgroups.android;
+
+public interface ObservableTagFactory {
+  String tag(Class<?> klass);
```

Comment:
Since the `AutoResubscribe` annotation uses a `Class<?>` then we can't use arbitrary `Object` here since that class is the one used here to determine the tag.